app/Console/Commands/InstallCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'council:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simplify installation process';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->intro();
+
+        if (!file_exists('.env')) {
+            exec('mv .env.example .env');
+            $this->line("\r\n.env file successfully created\r\n");
+        }
+        
+        exec('composer install');
```

Comment:
Keep in mind that this command requires `Illuminate` files, so if we're running this, we can assume this has already been done 😄

Reply:
Damn, you're right. :)